helloworld/notebooks/README.md

```diff
@@ -0,0 +1,94 @@
+# Transmogrify on Jupyter
+
+In this section we will look at how Transmogrify can be run within Scala notebooks on
+Jupyter.
+
+We are going to leverage [BeakerX](http://beakerx.com/) scala kernel for Jupyter
+
+## Setup BeakerX on Linux / Ubuntu
+
+Prequisities :
+
+* Apache Maven
+* Python 3
+* JDK 8 (JDK 10 or above can cause issues with Transmogrify)
+
+Installation using pip
+
+```$xslt
+sudo pip install beakerx
+sudo beakerx install
+```
+
+Installation using conda
+
+```$xslt
+conda create -y -n beakerx 'python>=3'
+source activate beakerx
+conda config --env --add pinned_packages 'openjdk>8.0.121'
+conda install -y -c conda-forge ipywidgets beakerx
+```
+
+Reference : [BeakerX Documentation](http://beakerx.com/documentation)
+
+## Setup BeakerX on Mac with Docker
+
+BeakerX provides a [docker container image](https://hub.docker.com/r/beakerx/beakerx/) on docker hub. 
+
+Assuming your Transmogrify source code is downloaded at `/Users/rdua/work/github/rajdeepd/TransmogrifAI`. You can use 
+the following command to start the container. We need the directory above so that we can mount sample notebooks and dataset
+into the container using docker volumes.
+
+### Increase the RAM available to Docker container
+
+Increase the Memory available to docker containers from the docker UI as shown below
+
+
+![docker-settings][docker-settings]
+
+[docker-settings]: ./images/docker_memory_settings.png 
+
+```bash
+docker run -p 8888:8888 -v /Users/rdua/work/github/rajdeepd/TransmogrifAI/helloworld/notebooks:/home/beakerx/helloworld-notebooks \
+-v /Users/rdua/work/github/rajdeepd/TransmogrifAI/helloworld:/home/beakerx/helloworld beakerx/beakerx
```

Comment:
maybe just change example to $TransmografaiPATH

Reply:
done